pcap.c

```diff
@@ -3016,14 +3016,14 @@ pcap_list_datalinks(pcap_t *p, int **dlt_buffer)
 		**dlt_buffer = p->linktype;
 		return (1);
 	} else {
-		*dlt_buffer = (int*)calloc(sizeof(**dlt_buffer), p->dlt_count);
+		*dlt_buffer = (int*)malloc(p->dlt_count * sizeof(**dlt_buffer));
```

Comment:
Same as above, this change would introduce an unpleasant surprise.

Reply:
Not really, because memset